optimize/client/src/components/Processes/Processes.js

```diff
@@ -141,7 +141,10 @@ export function Processes({mightFail, user}) {
                 });
               }
 
-              if (user?.authorizations.includes('entity_editor')) {
+              if (
+                user?.authorizations.includes('entity_editor') &&
+                optimizeDatabase !== 'opensearch'
```

Comment:
Could you check on `elasticsearch` here instead of `opensearch`? `elasticsearch` is used elsewhere, it'd be nice to keep it consistent. 

And since the string will be used twice, can you move it to a reuseable constant?

So, this would become something like `optimizeDatabase === elasticSearchIdentifier`